src/client/components/copybutton/copybutton.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {Button} from 'reactstrap';
+import { copyToClipboard } from '../../../utils/utils';
+import PropTypes from 'prop-types';
+
+class CopyButton extends React.Component {
+    static propTypes = {
+        color: PropTypes.string.isRequired
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {color: this.props.color};
+    }
+
+    render() {
+        return (
+            <Button color={this.state.color} onClick={() => this.handleClick()}>
```

Comment:
Usually one would bind `handleClick` to `this` and pass the onClick-handler as `<Button onClick={this.handleClick} />`. Could you change it to that pattern?